frontend/src/components/Client/AddToCartButton/AddToCartButton.js

```diff
@@ -0,0 +1,9 @@
+const AddToCartButton = () => {
+  return (
+    <button className="bg-violet-400 text-white cursor-pointer w-28 h-8 rounded-[5px] md:w-32 md:text-xl">
```

Comment:
I think we should keep the button styles and color consistent (kinda like the login/register form buttons except not w-full).
We can use these tailwind utilities for this button:
shadow
bg-indigo-500
hover:bg-indigo-400
text-white
py-2 
px-3
uppercase
tracking-wide
font-semibold
rounded
focus:ring
focus:ring-indigo-300 
md:text-xl